modules/standard/Random.chpl

```diff
@@ -1735,7 +1732,7 @@ module Random {
           myStart += multiplier * ZD.indexOrder(((...outer), innerRange.lowBound)).safeCast(int(64));
         else
           myStart += multiplier * ZD.indexOrder(innerRange.lowBound).safeCast(int(64));
-        if !innerRange.stridable {
+        if innerRange.hasUnitStride() {
```

Comment:
Was there a discussion about the "recommended" way of doing this kind of check? Personally I kind of like ``innerRange.strides.isOne()`` instead of ``innerRange.hasUnitStride()``, but that's just me.